### DISCUSSION TITLE
Anyone else having issues getting PostgreSQL to work with persistence

### DISCUSSION THREAD

**Discussion by matanyall** [infrahq/infra#3555] [Q&A] [answered]
Hey everyone! 

Trying to get a self-hosted version of infrahq working on GKE, and I'm having all sorts of issues just getting a test up and running.

For some reason, running the recent helm chart install of `helm upgrade --install infra infrahq/infra --values infra-values.yaml` failed to get the server to start due to a failure of initdb to get postgres to start up due to this error from the statefulset logs: 

```
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

initdb: error: directory "/var/lib/postgresql/data" exists but is not empty
It contains a lost+found directory, perhaps due to it being a mount point.
Using a mount point directly as the data directory is not recommended.
Create a subdirectory under the mount point.
```

Looks like something is going wrong with how the mountpoints are set, and ext4 creates the lost+found dir. I was surprised by this error, as there is no way around it without changing how things work deep in the helm chart. Is this a bug, or is there something I am missing?

Also, is there some slack/discord community for y'all? I have questions, and I'm potentially looking to productionize infrahq for our internal use. Thanks for the help!

**Comment by mchiang0610** [comment]
Hey @MatanyaLoewenthal it looks like it's a bug on our end. We will be making a fix. Thanks for the catch. 

We don't yet have a public slack/discord. Is there an e-mail I can invite you to slack? ping me at michael -at- infrahq.com

**Reply by mchiang0610** [reply]
This fix https://github.com/infrahq/infra/pull/3556 by @mxyng should resolve this.

**Reply by matanyall** [reply]
Amazing, Thanks!
I'll try to try it out soon

**Comment by mchiang0610** [comment, marked as answer]
@MatanyaLoewenthal we released a new helm chart (v0.20.5) that includes a fix for this. Possible to confirm it is working on your end? 

`helm repo update infrahq`

**Reply by matanyall** [reply]
Looks to be working!

**Reply by mchiang0610** [reply]
Awesome to hear. Let me know how it goes. Happy to answer any questions
